### ISSUE TITLE
What is the status of this project?

### ISSUE THREAD

**Issue by cjke** [HubSpot/tether#252]
Great project, just curious on project status?

Quite a few open issues dating back to around 2015. It's cool if you guys aren't maintaining it anymore, but could you do an official handover or update the readme to mark it as not actively maintained?

**Comment by TrevorBurnham** [comment]
We're nominally still maintaining it, but it isn't a priority for us. If another organization expressed serious interest in taking over project maintenance, we'd definitely be interested in a handover.

Meanwhile, [Popper.js](https://popper.js.org/) is looking terrific, and I'd encourage developers looking for a more actively maintained library in the same space to check it out.

**Comment by cjke** [comment]
Cool, again totally fair - OSS can be pretty unrewarding - so want to say I completely understand. Would it be worth mentioning that on the readme? This is a first hit for google for this type of JS lib and lots of stackoverflow answers link through to it, which is how I ended up with it. 

After investing some time integrating it into our application, we've found that some of the open bugs and issues are also pain points for us. Now, this is entirely on my team, absolutely, so don't think I am having a shot. But, it would be nice to hit the github page, and see a message saying "not actively maintained" (and maybe even close the issue tracker?). It would allow people to make informed decisions.

**Comment by FezVrasta** [comment]
Thank you so much for the note! I'll do my best 👍 And thank you again for the great work you did with Tether.

**Comment by nathancahill** [comment]
@TrevorBurnham What would a handoff in project ownership/maintenance look like from the side of the HubSpot team?

**Comment by FezVrasta** [comment]
@nathancahill may you share the reasons for wanting this library to being maintained again instead of switch to Popper.js? I'd like to understand the areas of improvement.

Thank you!

**Comment by nathancahill** [comment]
There's a lot of existing code that uses Tether in the wild. Ideally, everything would transition to a maintained library like Popper.js. But that's unrealistic. So the second best option is to provide enough ongoing support to maintain Tether in a working condition. Simple things like #175 are just gathering dust in the PR backlog.

**Comment by FezVrasta** [comment]
Would a compatibility layer between the Tether API and Popper.js help in this case?

**Comment by nathancahill** [comment]
I'm not sure. Maybe if it was a drop-in replacement? Would Popper.js expose a `new Tether` compatibility API that translates Tether options to Popper options? Seems more complicated than maintaining Tether.

**Comment by FezVrasta** [comment]
Yes I was thinking about a drop-in replacement, you simply set an alias in your webpack config (or babelrc) to `tether: 'popper-tether-compat'` and you get a Tether compatible API that acts by bridge.

It wouldn't be that hard actually, it's just a matter of map each Tether option to the Popper.js equivalent.
